src/main/kotlin/com/exactpro/th2/common/schema/configuration/IConfigurationProvider.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.exactpro.th2.common.schema.configuration
+
+interface IConfigurationProvider {
+    fun <T : Any> load(configClass: Class<T>, alias: String, default: () -> T): T
```

Comment:
What if I don't have a default option?

We should have a method that can load required configurations